src/services/meta_transaction_service.ts

```diff
@@ -206,6 +216,11 @@ export class MetaTransactionService {
             )
             .callAsync();
 
+        // log quote report and associate with txn hash if this is an RFQT firm quote
```

Comment:
Does this mean we don't get a report if the meta TX didn't consider an RFQT quote

Reply:
right now all meta-txn calls are RFQT enabled b/c they all come from matcha. 

for non-meta-txn calls I _do_ limit it only to RFQT firm quotes (https://github.com/0xProject/0x-api/blob/ba495a78e305113478396539e74577f84e2f3fba/src/handlers/swap_handlers.ts#L67) since there will be a lot of verbose log lines emitted. 

if we find this data useful for firm quotes we can consider opening up to all requests (as long as get confirmation from Oskar that our log serve can handle it)